query/search.js

```diff
@@ -160,6 +162,10 @@ function getQuery(vs) {
 
 }
 
+function forceLibpostal(params) {
+  return _.isSet(params, 'force_libpostal');
```

Comment:
It's a very misleading method name.  I'll submit a PR to change the name to `isTotesASet`.